mlflow/utils/async_logging/run_operations.py

```diff
@@ -0,0 +1,54 @@
+class RunOperations:
+    """
+    Represents a collection of operations on one or more MLflow Runs, such as run creation
+    or metric logging.
+    """
+
+    def __init__(self, operation_futures):
+        self._operation_futures = operation_futures or []
```

Comment:
yes but in tracking_service_client, where we split the large batch into smaller batches and send those batches, it creates multiple run_operations, so instead of returning [RunOperations], i am merging all of them in one and returning single object which can be awaited by client. Let me know if you have different approach that would be suitable here.

Reply:
thanks for the explanation!